src/crypto/verification/Base.js

```diff
@@ -103,6 +117,21 @@ export default class VerificationBase extends EventEmitter {
         });
     }
 
+    /* send a message to the other participant, using in-roomm messages
+     */
+    _sendMessage(type, content) {
+        if (this._done) {
+            return Promise.reject(new Error("Verification is already done"));
+        }
+        // FIXME: only use one of m.relationship/m.relates_to, once MSC1849
+        // decides which one to use
+        content["m.relationship"] = content["m.relates_to"] = {
```

Comment:
I guess you're doing both here because of [this discussion](https://github.com/matrix-org/matrix-doc/pull/1849/files/fa7c338a42389bb378cd2e52c1e62654e7fb0c8d#r317411401)? I'd just stick with `"m.relates_to"` for now, as we're not doing this anywhere else, but feel free to leave it like this.